environment.yml

```diff
@@ -1,6 +1,7 @@
 channels:
   - conda-forge
 dependencies:
+  - scikit-geometry
```

Comment:
@wolfv should I add the version number ?

Reply:
I think this is fine! 

Sorry, I didn't even realize that this repository already had an environment.yml file :facepalm: